doc/TESTING.md

```diff
@@ -0,0 +1,35 @@
+# Continuous Integration Testing
```

Comment:
We have a lot of developers who fly by and dump PRs sometimes that are not thoroughly tested. I don't think it's a good idea to move this into a separate file, nor most of the other content. The README.md really isn't that large to begin with.